tests/test_provider_configuration.py

```diff
@@ -102,3 +218,9 @@ def test_copy_should_overwrite_existing_value(self):
         data = OIDCData(abc='xyz')
         copy_data = data.copy(qwe='rty', abc='123')
         assert copy_data == {'abc': '123', 'qwe': 'rty'}
+
+    def test_del_and_len(self):
```

Comment:
I don't think this test is needed since those methods are directly delegated to the underlying storage.
(That's why there's only the case for `OIDCData.__str__` above - it's the only method except copy which has some custom behavior.)

Reply:
It has increased the coverage.

Reply:
Same as above, it's unrelated to this PR and I really don't think it's necessary to test builtin dict-behavior.